### DISCUSSION TITLE
想用chat gpt 做一个小游戏，属于公益游戏，求教学，可以有偿

### DISCUSSION THREAD

**Discussion by 11990908** [SchBenedikt/Text-Editor#32] [Q&A] [answered]
简单的剧情游戏，单机游戏即可，没有代码基础的人如何用chat GPT 和其他AI平台根据自己的想法制作游戏呢。

**Comment by CelsiaSolaraStarflare** [comment, marked as answer]
I don't think this is related to our topic. 

If you already know to use ChatGPT for that then try it.
